src/App.js

```diff
@@ -11,9 +11,90 @@ import HugeAss from "./Pages/HugeAss/HugeAss";
 import Footer from "./Components/Footer/Footer";
 import Episodes from "./Pages/Episodes/Episodes";
 import Playback from "./Components/Playback/Playback";
+import axios from "axios";
+
+var XMLParser = require("react-xml-parser");
+
+function convertUnicode(myString) {
```

Comment:
it might be a good idea to build a little helper file that has all these helper functions in it and then import them into this file. Since you might be reusing them for the other xml feed as well

Reply:
Since you are doing this in a new branch I will resolve this for now